autotx/utils/ethereum/mock_erc20.py

```diff
@@ -312,5 +312,24 @@
 				],
 				"stateMutability": "nonpayable",
 				"type": "function"
-			}
+			},
```

Comment:
Can we move this to a WETH abi?
Also, we should probably rename this ABI to just erc2_abi